backend/api/Areas/Leases/Mapping/Lease/TermMap.cs

```diff
@@ -32,11 +32,11 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.TermRenewalDate, src => src.RenewalDate)
                 .Map(dest => dest.TermExpiryDate, src => src.ExpiryDate)
                 .Map(dest => dest.TermStartDate, src => src.StartDate)
-                .Map(dest => dest.LeaseTermStatusTypeCode, src => src.StatusTypeCode.Id)
-                .Map(dest => dest.LeasePmtFreqTypeCode, src => src.LeasePmtFreqTypeCode.Id)
-                .Map(dest => dest.IsGstEligible, src => src.IsGstEligible ? src.GstAmount : null)
-                .Map(dest => dest.IsTermExercised, src => src.StatusTypeCode.Id == LeaseTermStatusTypes.EXER)
-                .Map(dest => dest.GstAmount, src => src.GstAmount)
+                .Map(dest => dest.LeaseTermStatusTypeCode, src => src.StatusTypeCode != null ? src.StatusTypeCode.Id : null)
```

Comment:
No need to do it now since it will require refactoring, but passing the type vs the flattened type code

Reply:
I think this is actually the type code, just not named appropriately.

Reply:
oh right, is the Model->Entity part